Sources/MapboxMaps/Annotations/ViewAnnotationOptions.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import UIKit
+
+public struct ViewAnnotationOptions: Hashable {
```

Comment:
@macdrevx do we have to add documentation to this class or can we just have a link pointing to `MapboxCoreMaps.ViewAnnotationOptions`?

I see that in case of the `CameraOptions` only the constructor is documented, but even there I think it would be cleaner to point to the description of the parameters without duplicating them in the constructor's documentation.

Reply:
We should fully document these public APIs. Links to MapboxCoreMaps documentation doesn't work well currently, so we should assume folks will only read these docs. The docs for each public interface should largely be able to stand on their own to minimize how much jumping around folks need to do when using the documentation.

Reply:
Alright, I'll add them. Thanks for the explanation!